src/container/plugins/FocusPlugin.js

```diff
@@ -246,8 +241,11 @@
 
 	plugin.teardown = function()
 	{
-		$(this.options.pauseFocusSelector).off('focus');
-		$(document).off('focus click', this._onDocClick);
+		document
+			.querySelector(this.options.pauseFocusSelector)
```

Comment:
Tagging this one too. May need to check that the query selection actually found something.